charts/charon/Chart.yaml

```diff
@@ -29,10 +29,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.2.8
+version: 0.3.0
```

Comment:
whichever way we decide consistency is needed I think

Reply:
if i understood correctly, this is `charon` chart, so there may be a possibility of operators running incompatible charts and running into a conflict.
But this is not the case with `charon-cluster` chart, as it is a standalone chart (more like a CDVC)

Reply:
but for the benefit of doubt, can make the versions consistent.